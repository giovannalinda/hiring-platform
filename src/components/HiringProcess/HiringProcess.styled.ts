import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  background: ${theme.colors.blue400};
  margin-top: 22.6rem;
  border-radius: ${theme.borderRadius.lg};
  text-align: center;
  padding: ${theme.space[7]};
`

export const TextGroup = styled.div`
  padding-top: 6.2rem;
  padding-bottom: 6.2rem;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;

  h1 {
    font-size: ${theme.font.sizes.title};
    color: ${theme.colors.white};
    padding-bottom: ${theme.space[7]};
  }

  p {
    color: ${theme.colors.blue100};
    font-size: ${theme.font.sizes.paragraph};
    max-width: 46.4rem;
  }
`

export const GridImage = styled.div`
  display: grid;
  grid-template-columns: 1fr 240px;
  grid-gap: ${theme.space[3]};
  width: 100%;
  padding: 0 6.2rem 6.2rem 6.2rem;

  div {
    display: grid;
    grid-template-rows: repeat(2, 1fr);
    grid-gap: ${theme.space[3]};
  }

  @media screen and (max-width: ${theme.breakpoints.tablet}) {
    display: flex;
    align-items: center;
    justify-content: center;
    flex-direction: column;
  }
`
