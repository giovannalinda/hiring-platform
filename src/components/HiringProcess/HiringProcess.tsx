import Image from 'next/image'
import { Assets } from '~/assets'

import * as S from './HiringProcess.styled'

export function HiringProcess() {
  return (
    <S.Container>
      <S.TextGroup>
        <h1>End-to-end hiring process</h1>
        <p>
          Redefining job application process and let you meet the best
          candidates in the shortest time
        </p>
      </S.TextGroup>

      <S.GridImage>
        <Image src={Assets.WomanWithHeadset} width={690} height={449} />

        <div>
          <Image src={Assets.ManWithHeadset} width={240} height={217} />
          <Image src={Assets.Woman} width={240} height={217} />
        </div>
      </S.GridImage>
    </S.Container>
  )
}
