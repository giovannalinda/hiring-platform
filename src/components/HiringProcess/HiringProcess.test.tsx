import { render } from '~/__helpers__/app-tests'
import { HiringProcess } from './HiringProcess'

describe('HiringProcess', () => {
  it('should match snapshot', () => {
    const { container } = render(<HiringProcess />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
