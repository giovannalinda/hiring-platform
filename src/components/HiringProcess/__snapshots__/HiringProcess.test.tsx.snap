// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HiringProcess should match snapshot 1`] = `
<section
  class="sc-bczRLJ iPZicK"
>
  <div
    class="sc-gsnTZi jnkmHM"
  >
    <h1>
      End-to-end hiring process
    </h1>
    <p>
      Redefining job application process and let you meet the best candidates in the shortest time
    </p>
  </div>
  <div
    class="sc-dkzDqf lbLRHs"
  >
    <img
      height="449"
      src="/WomanWithHeadset.png"
      width="690"
    />
    <div>
      <img
        height="217"
        src="/ManWithHeadset.png"
        width="240"
      />
      <img
        height="217"
        src="/Woman.png"
        width="240"
      />
    </div>
  </div>
</section>
`;
